### PR TITLE
[stable/selenium] Add configurable securityContext in all selenium deployments

### PR DESCRIPTION
**Pull request by DoroGi**
#### Is this a new chart
no

#### What this PR does / why we need it:
This adds the possibility to configure the security context in the Selenium deployments. I set the default user to 1000 as it's what is being used by the underlying image: https://github.com/SeleniumHQ/docker-selenium/blob/master/Base/Dockerfile. Not sure if this should be considered a breaking change or not. I hope a "patch" version bump is ok

#### Which issue this PR fixes
I didn't open an issue

#### Special notes for your reviewer:
This is my first PR to a public repo of this size. Please be patient if you find I made some mistakes.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/selenium/values.yaml [hunk 1]
```diff
@@ -53,6 +53,11 @@ hub:
   ## Path for checking readiness and liveness via HTTP Request
   probePath: "/wd/hub/status"
 
+  # Configure security context on the hub pod
```
Comment (diemol) [inline stable/selenium/values.yaml]: Do we need to leave these values as a default?
Can we leave them commented out and allow each person to enable them on their own?

Comment (DoroGi) [reply stable/selenium/values.yaml]: I am not actually sure about the best "non breaking change" default to put here.
The underlying image is using 1000 as a user, my understanding is that putting here anything else would be a breaking change. 
But I may be wrong. Let me know what you think.

**Comment by DoroGi** [comment]
Made a rebase and a version bump

**Comment by DoroGi** [comment]
Sorry, I had some problems rebasing.
Made a simple version bump.

**Pull request closed without merging**